include/bluetooth/l2cap.h

```diff
@@ -26,6 +26,12 @@ extern "C" {
 #include <bluetooth/conn.h>
 #include <bluetooth/hci.h>
 
+/* specific encryption errno values */
```

Comment:
L2CAP has nothing to do with these error/reasons... it defines its own to be used on-air/protocol. Move these to bluetooth.h?